src/components/Drawer.js

```diff
@@ -229,7 +231,8 @@ class Drawer extends React.Component {
       },
       content: {
         backgroundColor: StyleConstants.Colors.WHITE,
-        height: '100%'
+        overflow: 'auto',
+        height: `calc(100% - ${HEADER_HEIGHT})`
```

Comment:
I'm not sure how well supported `calc` is for mobile devices.  There is `?` for android, IE, and Opera.

https://developer.mozilla.org/en-US/docs/Web/CSS/calc